src/Illuminate/Database/Schema/MySqlSchemaState.php

```diff
@@ -50,8 +50,10 @@ protected function removeAutoIncrementingState(string $path)
      */
     protected function appendMigrationData(string $path)
     {
+        $migrationsTable = config('database.migrations', 'migrations');
```

Comment:
You can't call the foundation config helper here.

Reply:
On my semi-live production test it just worked. 
Would `app('config')->get('database.migrations')` work here?

Reply:
It'll work because you're using the database component with the Laravel framework. People building other implementations with the database component won't be able to do so anymore because we're hardwiring the framework in this way.

Reply:
Well, in that case I don't have a solution for this issue, and we'd better close this PR.

Reply:
what do you suggest instead? using command parameter instead of the config?

Reply:
In this case the `SchemaState` object should receive the table name as a constructor argument. Could be done by adding an extra argument with a default fallback to `migrations`. Ideally it would be passed through with the `getSchemaState` method but that'd be a breaking change if we added the parameter to that method signature. I'm thinking this will best be solved in the next major release but I understand that this is a bug fix technically. Not sure what the pragmatic way to solve this would be.